jobs/gcp-discovery/src/main/java/com/tmobile/pacbot/gcp/inventory/collector/APIKeysInventoryCollector.java

```diff
@@ -0,0 +1,92 @@
+package com.tmobile.pacbot.gcp.inventory.collector;
+
+import com.google.api.apikeys.v2.ApiKeysClient;
+import com.google.api.apikeys.v2.Key;
+import com.google.gson.Gson;
+import com.tmobile.pacbot.gcp.inventory.auth.GCPCredentialsProvider;
+import com.tmobile.pacbot.gcp.inventory.vo.APIKeysVH;
+import com.tmobile.pacbot.gcp.inventory.vo.ProjectVH;
+import net.minidev.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+
+import java.util.*;
+
+@Component
+public class APIKeysInventoryCollector {
+
+    @Autowired
+    GCPCredentialsProvider gcpCredentialsProvider;
+    private static final Logger logger = LoggerFactory.getLogger(APIKeysInventoryCollector.class);
+    private Object value;
+
+    public List<APIKeysVH> fetchApiKeys(ProjectVH projectVH) throws Exception {
+        String parent = "projects/"+projectVH.getProjectId()+"/locations/global";
```

Comment:
Is this applicable to only global locations?

Reply:
yes Dheeraj